src/EFTools/EntityDesign/VisualStudio/ModelWizard/gui/WizardPageDbConfig.cs

```diff
@@ -676,8 +676,17 @@ private void UpdateSettingsFromGui()
 
             // these connection strings & invariant names are coming from the ddex provider, so these are "design-time"
             var invariantName = DataConnectionUtils.GetProviderInvariantName(_dataProviderManager, _dataConnection.Provider);
+            var fixedDecryptedConnectionString = ReplaceMdsKeywords(decryptedConnectionString);
+            var fixedAppConfigConnectionString = ReplaceMdsKeywords(appConfigConnectionString);
             Wizard.ModelBuilderSettings.SetInvariantNamesAndConnectionStrings(ServiceProvider,
-                Wizard.Project, invariantName, decryptedConnectionString, appConfigConnectionString, true);
+                Wizard.Project, invariantName, fixedDecryptedConnectionString, fixedAppConfigConnectionString, true);
+
+            string ReplaceMdsKeywords(string connectionString)
+            {
+                connectionString = connectionString.Replace("Multiple Active Result Sets=", "MultipleActiveResultSets=");
```

Comment:
Can we include a comment here about why we need to do this?

Reply:
I'm gonna do that in a follow-up PR to avoid a re-spin since it's comment only, good call though.